pygeo/parameterization/DVGeoCST.py

```diff
@@ -1185,15 +1224,19 @@ def computeCSTfromCoords(xCoord, yCoord, nCST, N1=0.5, N2=1.0, dtype=float):
         # Normalize x and y
         chord = np.max(xCoord) - np.min(xCoord)
         xCoord = (xCoord - np.min(xCoord)) / chord
-        yCoord /= chord
+        yCoord = yCoord / chord
+        yTE = yTE / chord
+
+        # Subtract the linear TE thickness function
+        yMinusTE = yCoord - yTE * xCoord
 
         # Compute the coefficients via linear least squares
         dydw = DVGeometryCST.computeCSTdydw(xCoord, N1, N2, np.ones(nCST), dtype=dtype)
-        w = np.linalg.lstsq(dydw.T, yCoord, rcond=None)[0]
+        w = np.linalg.lstsq(dydw.T, yMinusTE, rcond=None)[0]
         return w
 
     @staticmethod
-    def plotCST(upperCoeff, lowerCoeff, N1=0.5, N2=1.0, nPts=100, ax=None, **kwargs):
+    def plotCST(upperCoeff, lowerCoeff, yUpperTE, yLowerTE, N1=0.5, N2=1.0, nPts=100, ax=None, **kwargs):
```

Comment:
Could we make these keyword arguments that default to zero to maintain backward compatibility?